src/points/types.ts

```diff
@@ -61,3 +66,10 @@ export interface GetHistoryResponse {
   hasNextPage: boolean
   nextPageUrl: string
 }
+
+export interface HistoryCardMetadata {
```

Comment:
i think that any types that are specifically for presentation should live with the presentational component (they're unlikely to be reused outside of that presentational context). it'd be nice for the types file to contain data types only?

Reply:
yeah, this is needed by both `cardDefinitions` and `PointsHistoryBottomSheet`, so i've moved it into `cardDefinitions` as a compromise